catalog/spatial/geocoding/spatial-geocoding-extract/src/main/java/org/codice/ddf/spatial/geocoding/extract/GeoNamesFileExtractor.java

```diff
@@ -51,6 +52,9 @@
 
   private String url;
 
+  private String[] schemes = {"http", "https", "ALLOW_ALL_SCHEMES"};
```

Comment:
❓How does this `ALLOW_ALL_SCHEMES` work? Should it be an option like `UrlValidator.ALLOW_LOCAL_URLS`? Why do you specify `ALLOW_ALL_SCHEMES` and also http/https?